acl_loader/main.py

```diff
@@ -583,6 +586,41 @@ def convert_input_interface(self, table_name, rule_idx, rule):
 
         return rule_props
 
+    def is_db_rule_dynamic(self, key):
+        try:
+            data = self.configdb.get_entry(self.ACL_RULE, key)
+            if 'is_time_based' in data and data['is_time_based'].lower() == 'true':
+                return True
+            return False
+        except Exception as e:
+            return False
+        
+    def is_time_based_rule(self, rule):
+        return rule.time_range.config.start_time
```

Comment:
We'd better check if `end_time` provided here

Reply:
Accept and thanks!